tools/include/endian.h

```diff
@@ -10,6 +10,14 @@
 #define bswap_16(x) OSSwapInt16(x)
 #define bswap_32(x) OSSwapInt32(x)
 #define bswap_64(x) OSSwapInt64(x)
+#define be16toh(x) OSSwapBigToHostInt16(x)
+#define htobe16(x) OSSwapHostToBigInt16(x)
+#define be32toh(x) OSSwapBigToHostInt32(x)
+#define htobe32(x) OSSwapHostToBigInt32(x)
+#define htole64(x) OSSwapHostToLittleInt64(x)
+#define htobe64(x) OSSwapHostToBigInt64(x)
+#define le64toh(x) OSSwapLittleToHostInt64(x)
+#define be64toh(x) OSSwapBigToHostInt64(x)
```

Comment:
@nbd168  has similar changes in his tree, see: https://git.openwrt.org/?p=openwrt/staging/nbd.git;a=commitdiff;h=33e84e8eef15d0eb13520669963af7972cdf81cb
https://git.openwrt.org/?p=openwrt/staging/nbd.git;a=shortlog

@nbd168  could you please have a look at this.

Reply:
Looks fine, applied.